transitiveDeps.md

```diff
@@ -0,0 +1,94 @@
+# Dependency Management
+
+The Azure SDK for JS client libraries include a host of dependencies, both internal and external. In general, our policy allows for some flexibility in dependencies and we generally pin to a [caret][caret] version - allowing for patches, security fixes, and any non-breaking changes in our dependency ranges.
+
+There are times when a dependency needs to be updated, such as when a security vulnerability has been found and patched in the depdendent codebase. While we will address security vulnerabilities in dependencies by updating our minimum version to the latest patched version, our semver policy allows customers to update to the latest version of a transitive dependency without waiting for our next releases and without requiring a hotfix. In this document we'll outline a few options available to you when a transitive dependency must be updated.
```

Comment:
In here, I read "server" instead of "semver" at a first glance, capitalizing would help to avoid confusion of a typo.

Also, I would suggest separating the last sentence in a different paragraph to improve readability and make the purpose of this document more visible.

```suggestion
There are times when a dependency needs to be updated, such as when a security vulnerability has been found and patched in the depdendent codebase. While we will address security vulnerabilities in dependencies by updating our minimum version to the latest patched version, our SemVer policy allows customers to update to the latest version of a transitive dependency without waiting for our next releases and without requiring a hotfix. 

In this document we'll outline a few options available to you when a transitive dependency must be updated.
```

Reply:
Or we could also say "Semantic Versioning policy" because I had to search online what "semver" means 😅